packages/labs/observers/package.json

```diff
@@ -168,6 +168,7 @@
     "@types/trusted-types": "^2.0.2"
   },
   "dependencies": {
+    "lit-html": "^2.4.0",
```

Comment:
Semver should at least include Lit 3
```suggestion
    "lit-html": "^2.4.0 || ^3.0.0",
```
But I wonder if it's worth it to create a Node build for this package instead of adding this dependency?

Reply:
Also needs to update package-lock.json to reflect the added dependency for `npm ci` to work.

Reply:
We need to do one of these, so I'm going to un-approve.

Reply:
Updated package and package-lock files to include lit-html 3.x